test/Controller/Plugin/ForwardTest.php

```diff
@@ -237,6 +238,35 @@ function ($e) {}
         $this->assertEquals(['content' => 'ZendTest\Mvc\Controller\TestAsset\ForwardController::testAction'], $result);
     }
 
+    public function testProblemListenersAreDetachedAndReattachedWhenPluginDispatchsRequestedController()
```

Comment:
No assertion or expected exception?

Reply:
@froschdesign Just bumping this, it's been 3 months and I'd love to start using ZF3. This issue is the only thing holding us back. 

I'm happy to make whatever change to the test. As my comment above says, my assertion is implied in the mock `once` invocation expectation. Would you prefer I change this to an explicit test spy? I'm not sure what the ZF test expectation would be for this use case.